aiorussound/util.py

```diff
@@ -69,3 +69,12 @@ def get_max_zones(model: str) -> int:
     if model in ("MCA-66", "MCA-C3"):
         return 6
     return 1
+
+
+def get_max_zones_favorites(model: str) -> int:
```

Comment:
This looks good, please add appropriate tests. Pretty identical to the method above so you can copy the test from there. 

https://github.com/noahhusby/aiorussound/blob/master/tests/test_util.py

Reply:
Noticed I missed the MBX models, so I added those here and in the new util test method